test/functional/test_storage.py

```diff
@@ -191,20 +193,20 @@ def test_controller_with_drive2(self):
             "max_sge": 128,
             "max_drive_per_controller": 6,
             "drives": drive2
-            }]
+        }]
 
         self.conf['compute']['storage_backend'].insert(0,
                                                        {
                                                            "type": "ahci",
                                                            "max_drive_per_controller": 6,
                                                            "drives": [
-                                                                {
-                                                                    'file': fixtures.image,
-                                                                    'bootindex': 1,
-                                                                    'use_msi': 'true',
-                                                                    'size': 8
-                                                                }
-                                                            ]
+                                                               {
```

Comment:
good catch

Reply:
IDE performs style reformation automatically for me, not me.